### DISCUSSION TITLE
Is there Buzzer API?

### DISCUSSION THREAD

**Discussion by meteozond** [zepp-health/zeppos-forum#172] [Developer Feedback]
It there any way to make sound with builtin buzzer?

**Comment by MyoungXUE** [comment]
Hi there,

Thanks for the great suggestion about adding more capabilities to the  Zepp OS ecosystem. Currently we do not have buzzer APIs available for Mini Programs, I will pass the feedback to our relevant teams.

Regards,
Zepp OS Team
